{"name": "test_user_list", "status": "failed", "statusDetails": {"message": "AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nassert None is not None", "trace": "@pytest.fixture(scope=\"session\", autouse=True)\n    def init_token():\n        \"\"\"初始化环境配置\"\"\"\n        cfg = read_yaml(\"config/config.yaml\")\n>       assert cfg is not None, \"配置文件 config/config.yaml 读取失败，请检查路径或内容\"\nE       AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nE       assert None is not None\n\nconftest.py:9: AssertionError"}, "start": 1761032991594, "stop": 1761032991594, "uuid": "f11b0603-de8f-4cbd-a431-48f73266144b", "historyId": "029a05ce26fea6b1a4a8b61bda7814a5", "testCaseId": "029a05ce26fea6b1a4a8b61bda7814a5", "fullName": "testcases.test_user_list#test_user_list", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_user_list"}, {"name": "host", "value": "DESKTOP-UGQEI6T"}, {"name": "thread", "value": "17608-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_user_list"}], "titlePath": ["testcases", "test_user_list.py"]}